opm/flowdiagnostics/TracerTofSolver.cpp

```diff
@@ -305,13 +305,14 @@ namespace FlowDiagnostics
         for (const auto& conn : g_reverse_.cellNeighbourhood(cell)) {
             const int upwind_cell = conn.neighbour;
             const double flux = conn.weight;
-            upwind_tof_contrib += tof_[upwind_cell] * flux;
+            upwind_tof_contrib += tof_[upwind_cell] * (tracer_[upwind_cell] != 0.0 ? tracer_[upwind_cell] : 1.0) * flux;
```

Comment:
Do we ever compute negative tracer concentrations?  If not, then this condition should probably be
```C++
tracer_[upwind_cell] > 0.0
```